### PR TITLE
feat(Packages):  sort by updatable packages by default

### PR DESCRIPTION
**Pull request by mkholjuraev**
resolves: https://issues.redhat.com/browse/SPM-651

### CODE REVIEW

File: src/Messages.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys */
```
Comment (jiridostal) [inline src/Messages.js:1]: Hehe, don't disable this one :-)

**Review by jiridostal — CHANGES_REQUESTED** [review]

File: src/Messages.js (hunk 1 above)
Comment (mkholjuraev) [reply src/Messages.js:1]: Actually, this line was supposed to be deleted ::)

**Comment by jiridostal** [comment]
Rebase needed :-(

**Pull request merged**